### ISSUE TITLE
incorrect highlightning of italic text in tmux, introduced in commit: f3ffe0b32

### ISSUE THREAD

**Issue by ilAYAli** [neovim/neovim#12193]
- `nvim --version`:
NVIM v0.5.0-471-gfc98f2d58
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/petter/source/neovim/build/config -I/home/petter/source/neovim/src -I/home/petter/source/neovim/.deps/usr/include -I/usr/include -I/home/petter/source/neovim/build/src/n
vim/auto -I/home/petter/source/neovim/build/include
Compiled by petter@pwa-lnx

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/home/petter/local/share/nvim"

- `OS`: macOS/linux
- `$TERM`: screen-256color
- `tmux version`: tmux 2.9a 

### Steps to reproduce:
@erw7 
I have not investigated why the empty `plug` settings are needed to trigger the bug.

_/tmp/simple.vimrc:_
```
call plug#begin('~/.vim/plugged')
call plug#end()
" this is a comment

hi! Comment ctermfg=14 gui=italic guifg=#888888
set termguicolors
```

`nvim -u /tmp/simple.vimrc /tmp/simple.vimrc`

### Actual behaviour
![Screenshot 2020-04-27 at 12 31 42](https://user-images.githubusercontent.com/1106732/80362608-36d14880-8883-11ea-923b-1a4289036eb6.png)

### Expected behaviour
Ignore the fact that the call to Plug was removed. It was just to _not_ trigger the bug, and show that the comment is correctly highlighted

![Screenshot 2020-04-27 at 12 32 09](https://user-images.githubusercontent.com/1106732/80362587-2de07700-8883-11ea-84ca-edea1d698112.png)

**Comment by pappasam** [comment]
I confirm that I have this issue as well (using the latest nightly release).

**Comment by erw7** [comment]
I think this will be corrected by #12199.

**Comment by erw7** [comment]
I don't know what plug is doing, but I think the minimal `init.vim` that would cause this to happen without plug should be as follows:

```vim
" this is a commen

autocmd ColorScheme * hi! Comment ctermfg=14 gui=italic guifg=#888888
set termguicolors
colorscheme default
```
If you don't call `colorscheme` in `init.vim`, I think `$VIMRUNTIME/syntax/synload.vim` will be called in `$VIMRUNTIME/syntax/syncolor.vim` to override the highlights set in `init.vim`.
